src/test/resources/testData/productCreationTestData.csv

```diff
@@ -0,0 +1,4 @@
+id, name, description, price, category_id
+10, Cross-Back Training Tank, The most awesome phone of 2013!, 299.0, 10
```

Comment:
CSV file shouldn't contain spaces between elements, like the productTestData.csv